File: MPI/mpicheck/tests/functions/2.0/MPI_Publish_name.f

        
        program main
        include 'mpif.h'
        
        CHARACTER*(10) var_0
       INTEGER var_1
       CHARACTER*(10) var_2
       INTEGER var_3
        call mpi_publish_name(var_0, var_1, var_2, var_3)
       call pmpi_publish_name(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Publish_name.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const char *var_0;
    MPI_Info var_1;
    const char *var_2;
    int ret;
    /* calls */
    ret = MPI_Publish_name(var_0, var_1, var_2);
    ret = PMPI_Publish_name(var_0, var_1, var_2);
    return 0;
}
